awd/dynamodb.py

```diff
@@ -21,7 +21,7 @@ def add_doi_item_to_database(self, doi_table, doi):
             TableName=doi_table,
             Item={
                 "doi": {"S": doi},
-                "status": {"S": "Processing"},
+                "status": {"S": "2"},
```

Comment:
Is there a way to use enum value here?

Reply:
No, importing the enum from `cli.py` creates a circular import issue. I can add a comment if that would help

Reply:
Could you move the enum from cli.py to a separate file since the special status value is shared across multiple classes/files ?

Reply:
Done

Reply:
Thanks, but the line still has the reference to number "2" (status enum not referenced).

Reply:
Whoops, my bad, fixed